commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -1,6 +1,7 @@
 package org.exoplatform.commons.dlp.service.impl;
```

Comment:
Modifications in this file should be removed I think

Reply:
Done!